app/retry.js

```diff
@@ -0,0 +1,16 @@
+const { totalRetries } = require('./config/publish')
+
+const retry = async (fn, retriesLeft = totalRetries, interval = 1000, exponential = true) => {
```

Comment:
Consider setting interval value in config file.

Reply:
Agreed.  Done